UKHomeOffice/sgmr-service
# SPCR-532 Create a submit component

### AC / Description of changes
In preparation for needing different submit CTA's on voyage form pages depending on whether you're on a change journey or a new journey.

- create a Submit component to handle CTA button display and actions
- update Voyage Departure Form to use new component
- update Voyage Arrival Form to use new comopnent


### To Test

- create a voyage
- fill in departure page
- save and come back later
- > should go to dashboard and see voyage in Drafts

- create a voyage
- fill in departure page
- save and continue
- fill in arrival page
- save and come back later
- > should go to dashboard and see voyage in Drafts


- create a voyage
- fill in departure page
- save and continue
- fill in arrival page
- save ad continue
- > should see pc page


### Notes

---

**Remember to merge to feature branches not `master`.**
